### PR TITLE
Update `Kudosu`

### PR DESCRIPTION
**Pull request by kevlu8**
Added:
- When kudosu should be given
- The usage of kudosu in BN applications

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)\

### CODE REVIEW

File: wiki/Modding/Kudosu/en.md
Comment (Walavouchey) [inline wiki/Modding/Kudosu/en.md:9]:
```suggestion
Kudosu is awarded to mod posts that are helpful and point out useful suggestions for the beatmap. If a mapper believes that a mod post is useful and addresses a valid issue, they can click the thumbs up button and award kudosu to the user who posted the mod. 
```

File: wiki/Modding/Kudosu/en.md
Comment (Walavouchey) [inline wiki/Modding/Kudosu/en.md:11]:
```suggestion
In [forum modding](/wiki/Modding/Forum_modding), kudosu used to be exchangeable for [star priority](/wiki/Modding/Star_priority).
```

File: wiki/Modding/Kudosu/en.md
Comment (Walavouchey) [inline wiki/Modding/Kudosu/en.md:13]:
```suggestion
Nowadays, kudosu is used for the evaluation of a user's capability to become a [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators). A user must have 200 kudosu in osu! or 150 kudosu in osu!taiko, osu!catch, or osu!mania in order to apply.
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Modding/Kudosu/en.md
Comment (cl8n) [reply wiki/Modding/Kudosu/en.md:11]: curious what u feel the difference is, these read identically to me

Comment (Walavouchey) [reply wiki/Modding/Kudosu/en.md:11]: slightly emphasises that it's not the case anymore

**Comment by Walavouchey** [comment]
for next time, read the [contribution guide](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits) and keep in mind the following:

- always make a new branch before making changes, don't pr from the master branch
- there's no need to update the branch with master yourself most of the time, since a maintainer will do that anyway before merging

other than that, ty for the pr!

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**
